redash/models.py

```diff
@@ -493,8 +493,7 @@ def to_dict(self, all=False, with_permissions_for=None):
                 DataSourceGroup.group == with_permissions_for,
                 DataSourceGroup.data_source == self).one()[0]
 
-        doc_url = self.options.get('doc_url', schema['properties'].get(
-            'doc_url', {}).get('default'))
+        doc_url = self.options.get('doc_url')
```

Comment:
Thanks @arikfr. I ended up having to wrap in a try/except, but this is much better.